src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,69 @@
-export const Pagination = () => {};
+import React from 'react';
+import cn from 'classnames';
+
+interface Props {
+  value: string;
+  column: number;
+  setColumn: (index: number) => void;
+  maxItems: number;
+}
+
+export const Pagination: React.FC<Props> = ({
+  value,
+  column,
+  setColumn,
+  maxItems,
+}) => {
+  const Columns = Math.ceil(maxItems / Number(value));
```

Comment:
```suggestion
  const columns = Math.ceil(maxItems / Number(value));
```
we usually name vars in camelCase